### PR TITLE
Adding Car Class

### PR DESCRIPTION
**Pull request by johnwon2007**

### CODE REVIEW

File: src/test/java/CarTest.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+    @Test(timeout = 50)
+    public void TestGetMaxSpeed() {
+        assertEquals(10, c.getMaxSpeed());
+    }
+
+    @Test(timeout = 50)
+    public void TestUpgradeSpeed() {
+        c.upgradeSpeed();
+        assertEquals(20, c.getMaxSpeed());
```
Comment (Polymatth) [inline src/test/java/CarTest.java:27]: Error assertEquals False please fix :)

**Review by Polymatth — CHANGES_REQUESTED** [review]
Everything looks good!

**Review by kdr47 — COMMENTED** [review]
error in cartest.java line 27

**Review by Polymatth — APPROVED** [review]
Everything looks great

**Pull request merged**